testing/go/create_extension_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2025 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package _go
+
+import (
+	"os"
+	"testing"
+
+	"github.com/dolthub/doltgresql/core/extensions"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+// SkipIfExtensionNotSupported skips if an extension (or set of extensions) is not installed locally. Missing extensions
+// aren't meaningful failures, since it relies on external factors. This can still cause failures due to differences
+// between versions, platforms, etc., but in that case we should write more generic tests (or use stable extensions) for
+// our testing purposes.
+func SkipIfExtensionNotSupported(t *testing.T, exts ...string) {
+	for _, ext := range exts {
+		_, err := extensions.GetExtension(ext)
+		if err != nil {
+			t.Skipf("Skipping test since extension `%s` is not installed", ext)
+		}
+	}
+}
+
+func TestCreateExtension(t *testing.T) {
+	if os.Getenv("CI") != "" {
+		t.Skip("figure out what to do with these in CI")
```

Comment:
Think I would probably call getting this running in CI a blocker. A docker container probably makes the most sense, similar to the replication tests.